GSA/digital-gov-drupal
# DIGITAL-60: add field to guide content type

## Jira ticket

[DIGITAL-60](https://cm-jira.usa.gov/browse/DIGITAL-60)

#Relates to

[Digital-43](https://github.com/GSA/digital-gov-drupal/pull/75)
[Digital-61](https://github.com/GSA/digital-gov-drupal/pull/80 )

## Purpose

Acceptance Criteria:

A dropdown text field that allows a single glossary vocabulary ID to be referenced

### QA/Testing instructions

1. Pull branch
2. Drush CIM
3. lando rebuild && lando si
4. Taxonomy module should be enabled.

1. Under the guide content type, make sure the field HCD Glossary is available and that it is set up to reference a vocabulary ID.

## Checklist for the Developer


## Checklist for the Peer Reviewers
